WordPress/Classes/Utility/WPError.m

```diff
@@ -71,53 +71,6 @@ + (void)showXMLRPCErrorAlert:(NSError *)error
 {
     NSString *message = nil;
     NSString *title = nil;
-
-    if ([error.domain isEqual:AFURLRequestSerializationErrorDomain] ||
-            [error.domain isEqual:AFURLResponseSerializationErrorDomain])
-    {
-        NSHTTPURLResponse *response = (NSHTTPURLResponse *)[error.userInfo objectForKey:AFNetworkingOperationFailingURLResponseErrorKey];
-        switch (error.code) {
-            case NSURLErrorBadServerResponse:
-                if (response) {
-                    switch (response.statusCode) {
-                        case 400:
-                        case 405:
-                        case 406:
-                        case 411:
-                        case 412:
-                        case 413:
-                        case 414:
-                        case 415:
-                        case 416:
-                        case 417:
-                            title = NSLocalizedString(@"Incompatible site", @"Error message shown in the set up process if the WP install was unable to be added to the app due to an error being returned from the site.");
```

Comment:
Are we able to keep these more informative error messages while ditching AFNetworking?

Reply:
Again these errors where not being displayed for a while (more than a year), since we moved the REST API classes to use Alamofire. 
We now handle this differently on the API classes on WordPressKit.

Reply:
Ah, I didn't realise that! :shipit: